xbmc/music/MusicDatabase.cpp

```diff
@@ -2077,12 +2090,36 @@ bool CMusicDatabase::GetArtist(int idArtist, CArtist& artist, bool fetchAll /* =
     }
     m_pDS->close(); // cleanup recordset data
 
+    artist.videolinks.clear();
+    if (fetchAll)
+    {
+      strSQL = PrepareSQL("SELECT idSong, strTitle, strMusicBrainzTrackID, strVideoURL, url "
+                          "FROM song JOIN album_artist ON song.idAlbum = album_artist.idAlbum "
+                          "LEFT JOIN art ON art.media_id = song.idSong AND art.type = 'videothumb' "
+                          "WHERE album_artist.idArtist = %i AND "
+                          "song.strVideoURL is not NULL GROUP by song.strVideoURL ORDER BY idSong",
+                          idArtist);
+      debugSQL += strSQL;
+      m_pDS->query(strSQL);
+      while (!m_pDS->eof())
+      {
+        const dbiplus::sql_record* const record = m_pDS->get_sql_record();
+        ArtistVideoLinks videoLink;
+        videoLink.title = record->at(1).get_asString();
+        videoLink.mbTrackID = record->at(2).get_asString();
+        videoLink.videoURL = record->at(3).get_asString();
+        videoLink.thumbURL = record->at(4).get_asString();
+
+        artist.videolinks.emplace_back(std::move(videoLink));
+        m_pDS->next();
+      }
+      m_pDS->close();
+    }
+
     auto end = std::chrono::steady_clock::now();
     auto duration = std::chrono::duration_cast<std::chrono::milliseconds>(end - start);
 
-    CLog::Log(LOGDEBUG, LOGDATABASE, "{0}({1}) - took {2} ms", __FUNCTION__, strSQL,
-              duration.count());
-
+    CLog::LogF(LOGDEBUG, "MusicDatabase::GetArtist - {} - took {} ms", debugSQL, duration.count());
```

Comment:
LogF already logs the method name

Reply:
Sure.  Just used to writing what I'd like to see in any logs.